packages/project-management-automation/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+	"extends": "../../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": "lib",
+		"declarationDir": "build-types",
+
+		// This is required due to a type error coming from missing types in @actions/github
```

Comment:
🤔

Reply:
Noted here in passing: https://github.com/WordPress/gutenberg/pull/18942#issuecomment-601793927

> There's a new error from `project-management-automation` which is using stricter rules (`noImplicitAny`) now. Seems to be the same as this, which was fixed: [actions/toolkit#199](https://github.com/actions/toolkit/issues/199). Is the `@actions/github` dependency up to date?

I haven't investigated and I'd like to leave it outside the scope here. `noImplicitAny` was disabled before which is why the error hadn't manifested.